Parser/Functions/MemoryAccessorFunction.cs

```diff
@@ -107,14 +107,6 @@ protected ParseErrorExpression BuildTrigger(TriggerBuilderContext context, Inter
                     {
                         if (mathematic.Operation == MathematicOperation.Subtract)
                             offsetConstant = new IntegerConstantExpression(-offsetConstant.Value);
-
-                        if (offsetConstant.Value < 0)
-                        {
-                            // Negative relative offsets can actually be handled by the runtime through overflow
-                            // addition, but the editor generates an error if the offset is larger than the
-                            // available memory space for the current system.
```

Comment:
This comment is very informative. It looks like that behavior was not duplicated when the editor was rewritten in 0.79.